packages/types/types/config/index.d.ts

```diff
@@ -1,6 +1,8 @@
 import { Config as ValidatorConfig } from '../validation/config';
+import { Report } from '../custom-report';
 
 export type Config = {
   silent?: boolean;
   validate?: ValidatorConfig;
+  reports?: Report<unknown, unknown>[];
```

Comment:
Maybe `generate.reports`?

Reply:
mb `generate.customReports`? So that it matches command line arguments